engine/src/cmd/unit_generic.cpp

```diff
@@ -2145,316 +2023,7 @@ static std::string NearestSystem( std::string currentsystem, QVector pos )
     return closest_system;
 }
 
-void Unit::UpdatePhysics( const Transformation &trans,
```

Comment:
I believe I changed some of this same code in #194 . We'll have to get my fixes into the new location of this code, whatever file it's in now.

Reply:
Yep. That was a somewhat painful merge...